Detectors/MUON/MID/Raw/include/MIDRaw/GBTBareDecoder.h

```diff
@@ -0,0 +1,68 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MIDRaw/GBTBareDecoder.h
+/// \brief  MID GBT decoder without user logic
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   12 March 2020
+#ifndef O2_MID_GBTBAREDECODER_H
+#define O2_MID_GBTBAREDECODER_H
+
+#include <cstdint>
+#include <array>
+#include <vector>
+#include <gsl/gsl>
+#include "DataFormatsMID/ROFRecord.h"
+#include "MIDRaw/CrateParameters.h"
+#include "MIDRaw/ELinkDecoder.h"
+#include "MIDRaw/GBTOutputHandler.h"
+#include "MIDRaw/LocalBoardRO.h"
+
+namespace o2
+{
+namespace mid
+{
+class GBTBareDecoder
+{
+ public:
+  void init(uint16_t feeId, uint8_t mask, bool isDebugMode = false);
+  void process(gsl::span<const uint8_t> bytes, uint16_t bc, uint32_t orbit, uint16_t pageCnt);
+  /// Gets the vector of data
+  const std::vector<LocalBoardRO>& getData() const { return mOutputHandler.getData(); }
+
+  /// Gets the vector of data RO frame records
+  const std::vector<ROFRecord>& getROFRecords() const { return mOutputHandler.getROFRecords(); }
+
+  bool isComplete() const;
+
+  /// Clears the decoded data
+  inline void clear() { mOutputHandler.clear(); }
```

Comment:
Well I wanted to make sure that it is inline, so that one saves time. But if I understand correctly what's written [here](https://en.cppreference.com/w/cpp/language/inline), it is probably implicitly inline.
I can remove it if needed.